domain-xample/docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: '3.9'
```

Comment:
Should this file and the other be nested in a src/docker dir to match with other uses in the repo?

Reply:
I considered that but I decided that the `src/docker` folder will be for building the Docker image of subprojects, while the `docker-compose.yml` belongs in the `domain-*` folder.  I don't anticipate more than one `docker-compose.yml` per domain. I plan to move `app/src/docker/docker-compose.yml` to `app` or the root folder. Open to feedback.